### PR TITLE
Updating CoreFx dependencies to beta-devapi-24410-02

### PR DESCRIPTION
**Pull request by krwq**
Attempt to fix the auto merge from dotnet-bot

### CODE REVIEW

**Comment by krwq** [comment]
@dotnet-bot test Innerloop Windows_NT Release Build and Test please

**Comment by krwq** [comment]
@dotnet-bot test Innerloop Windows_NT Debug Build and Test please

File: src/Common/test-runtime/project.json [hunk 1]
```diff
@@ -8,6 +8,8 @@
     "System.IO.Compression": "4.2.0-beta-devapi-24410-02",
     "System.Runtime.InteropServices.RuntimeInformation": "4.0.1-beta-devapi-24410-02",
     "System.Linq.Parallel": "4.0.2-beta-devapi-24410-02",
+    "System.Private.Xml": "4.0.0-beta-devapi-24410-02",
+    "System.Private.Xml.Linq": "4.0.0-beta-devapi-24410-02",
```
Comment (joperezr) [inline src/Common/test-runtime/project.json]: Do you mind creating an issue to remove these two once System.Xml.ReaderWriter and System.Xml.XDocument dependencies get fixed?

**Comment by joperezr** [comment]
LGTM assuming CI passes.

**Pull request merged**